servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSResources.java

```diff
@@ -38,6 +41,9 @@ public Domain getDomain(@PathParam("domain") String domain) {
                 System.err.println("*** Warning: undeclared exception (" + code + ") for resource getDomain");
                 throw typedException(code, e, ResourceError.class);
             }
+        } finally {
+            this.delegate.stopTiming(timerMetric, "GET", code, "getDomain", context);
```

Comment:
since the code is auto generated I'm guessing the generator needs to include the "getDomain" here and in the next line automatically. otherwise we can just set the method name in the resource context as well.

Reply:
Yes I assumed that while way we generate the function name in ZMSResources we can put it here